test/test_apps/5.0/Appraisals

```diff
@@ -0,0 +1,40 @@
+case(RUBY_VERSION)
+when "2.2.5" then
+
+  appraise "capybara-2.2-ruby#{RUBY_VERSION}" do
```

Comment:
> So many capybaras...are they all necessary?

I think we did it on purpose base on this https://github.com/appfolio/ae_page_objects/blob/master/development.md#integration

> On the other hand, there are newer versions as well.

yea, maybe we should have separate PR for adding new capybara support